src/auth_components/Login.tsx

```diff
@@ -26,8 +26,9 @@ export default function Login({
       await login(emailRef.current.value, passwordRef.current.value)
     } catch(error) {
       console.log(error.code)
+      const alteredMessage = error.message === 'The password is invalid or the user does not have a password.'? 'Wrong password, please try again.' : error.message
```

Comment:
You might want to consider using `error.code` to decide to generate a friendly message, just in case firebase ever decides to change the content of this `message` string. Which `error.code` is it in this case?

Reply:
oooh, good point! the error code is "auth/wrong-password". ill update the PR